app/controllers/submissions_controller.rb

```diff
@@ -946,6 +957,27 @@ def notebook_to_html(file_contents, unique_path, type)
     File.read(cache_file)
   end
 
+  def rmd_to_html(file_contents, unique_path)
+    cache_file = Pathname.new('tmp/rmd_html_cache').join("#{unique_path}.html")
+    unless File.exist? cache_file
+      FileUtils.mkdir_p(cache_file.dirname)
+      begin
+        file_contents.gsub!(/```{r[^}]*}/, '```r')
```

Comment:
tighten up this regex a little by using the start of line character, and accepting arbitrary whitespace before and after the triple backticks